src/signals/incident/definitions/wizard-step-3-telefoon.js

```diff
@@ -18,7 +18,7 @@ export default {
           subtitle: 'Zo kunt u ons helpen het probleem sneller of beter op te lossen.',
           path: 'reporter.phone',
           placeholder: 'Telefoonnummer',
-          type: 'text',
+          type: 'tel',
           autoRemove: /[^ ()\d+-]/g,
```

Comment:
What if someone would input their phone number like +31612345678? Would that result in `31612345678`?
The regex seems a bit strange to me; does it even do anything?

Reply:
on blur it will remove unwanted characters: "+" is allowed

Reply:
The `autoRemove` option doesn't seem to work. Can you confirm?

Reply:
again: on blur the invalid characters are removed. please look at telefoonnummer step

https://acc.meldingen.amsterdam.nl/incident/telefoon